pkg/ddc/alluxio/transform.go

```diff
@@ -356,6 +357,19 @@ func (e *AlluxioEngine) transformMasters(runtime *datav1alpha1.AlluxioRuntime,
 	if err != nil {
 		e.Log.Error(err, "failed to transform volumes for master")
 	}
+
+	if IsMountWithConfigMap() {
+		e.Log.Info("use configmap to generate mount info")
+		// transform mount secret options
+		nonNativeMounts, err := e.generateNonNativeMountsInfo(dataset)
+		if err != nil {
+			e.Log.Error(err, "generate non native mount info occurs error")
+			return err
+		}
+		value.Master.NonNativeMounts = nonNativeMounts
+		value.Master.MountConfigStorage = "configmap"
```

Comment:
This value "configmap" has been used in many places,can we  replaced the value "configmap" here by a constant？

Reply:
ok, done.